### PR TITLE
Api

### PR DESCRIPTION
**Pull request by therightmandev**

### CODE REVIEW

File: slack_wrapper/api.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+			return True
+	
+	def api_method(self, method, extra_params={}):
+		"""executes any method and returns a dictionary"""
+		params = {'token': self.TOKEN}
+		params.update(extra_params)
+		response_dict = json.loads(requests.get(
+                                        self.domain + 'channels.list',
+                                        params=params
+                                ).text)
```
Comment (raylu) [inline slack_wrapper/api.py:23]: why `json.loads(...` instead of just `requests.get(...).json()`?

File: slack_wrapper/api.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import requests, json
+
+class SlackAPI():
+	def __init__(self, token):
+		self.TOKEN = token
+		self.domain = 'https://slack.com/api/'
+
+	def has_error(self, response_dict, log="An error ocurred:"):
```
Comment (raylu) [inline slack_wrapper/api.py:8]: this function is... kind of bizarre and requires a lot of discipline to use. why not build in the functionality to api_method and have it actually raise an exception?

Comment (therightmandev) [reply slack_wrapper/api.py:8]: @raylu I'm planning on adding more functions like `get_channel_list` that will be used for more important API methods and `has_error` will be in each one of them.

File: slack_wrapper/api.py (hunk 1 above)
Comment (therightmandev) [reply slack_wrapper/api.py:23]: @raylu simple: I didn't know that the second option existed and did the same, I'll change that

File: slack_wrapper/api.py (hunk 2 above)
Comment (raylu) [reply slack_wrapper/api.py:8]: hence the "requires a lot of discipline to use" part

Comment (therightmandev) [reply slack_wrapper/api.py:8]: Oh, I didn't get what you meant by "why not build in the functionality to api_method and have it actually raise an exception?". Or the "requires a lot of discipline to use" for that matter. Yeah, I'll try to integrate it in api_method

**Pull request merged**